src/pages/ReimbursementAccount/PersonalInfo/substeps/Confirmation.tsx

```diff
@@ -115,7 +105,7 @@ function Confirmation({reimbursementAccount, reimbursementAccountDraft, onNext,
                         <DotIndicatorMessage
                             textStyles={[styles.formError]}
                             type="error"
-                            messages={{0: error}}
```

Comment:
Please test if this change creates regressions

Reply:
Its fine, we've applied this change in multiple places before